common/src/test/scala/uk/ac/wellcome/elasticsearch/WorksIndexTest.scala

```diff
@@ -34,27 +35,31 @@ class WorksIndexTest
   implicitly[Arbitrary[Work]]
 
   it("puts a valid work") {
-    createAndWaitIndexIsCreated(worksIndex, indexName)
 
-    val sampleWork = Arbitrary.arbitrary[Work].sample.get
-    val sampleWorkJson = JsonUtil.toJson(sampleWork).get
+    forAll{ sampleWork: Work =>
```

Comment:
This is because the test didn't initially spot that I needed to add the mapping for `label` in `WorksIndex`. The reason why it didn't fail is that it was randomly generating one instance of `Work` and if that happened to not have a `PhysicalLocation`, the test would still pass. 
So this is to make it try more than one instance of `Work`